package.json

```diff
@@ -27,16 +27,20 @@
     "@actions/github": "^5.1.1",
     "@actions/io": "^1.1.2",
     "@actions/tool-cache": "^2.0.1",
-    "semver": "^7.3.8"
+    "semver": "^7.3.8",
+    "sqlite3": "^5.1.6",
```

Comment:
Let me check if there is a reasonable way for us to limit this being installed in all workflows. Preferably, we isolate this as much as we can. But, if there is no way to do so, let's just keep it as is.

![image](https://github.com/expo/expo-github-action/assets/1203991/27c04537-e40a-4aa3-beca-08944af7e110)

Reply:
good catch! i think we don't really need the sqlite3 here. replace it by `@types/sqlite3`. but we still have the fixed `sqlite3@^5.1.6` in the code. that's the version we want to install dynamically in github actions (with cache supported)

Reply:
oh no, it will break unit test. well, let me fallback to add sqlite3 to devDependencies.